examples/hotrod/pkg/log/spanlogger.go

```diff
@@ -57,99 +58,164 @@ func (sl spanLogger) With(fields ...zapcore.Field) Logger {
 	return spanLogger{logger: sl.logger.With(fields...), span: sl.span, spanFields: sl.spanFields}
 }
 
-func (sl spanLogger) logToSpan(level string, msg string, fields ...zapcore.Field) {
-	// TODO rather than always converting the fields, we could wrap them into a lazy logger
-	fa := fieldAdapter(make([]log.Field, 0, 2+len(fields)))
-	fa = append(fa, log.String("event", msg))
-	fa = append(fa, log.String("level", level))
```

Comment:
Level was used!

Reply:
Ha! Yes, you're right, my bad for this oversight and thanks for the callout. Added "level" as an attribute to the logs in: [a31b067](https://github.com/jaegertracing/jaeger/pull/4556/commits/a31b0679180f7e62c1576ccbb2cfe7903d0d0da5)

Note that the MySQL spans are missing the log level:

<img width="1618" alt="Screenshot 2023-07-02 at 7 20 44 am" src="https://github.com/jaegertracing/jaeger/assets/26584478/88a85215-a997-4029-9fb6-55d6e4331100">

That's because `LogFields` is invoked directly from the active span here: https://github.com/jaegertracing/jaeger/blob/343fcc6ddd3bdb94dec864bddc2ff8794d6666a8/examples/hotrod/pkg/tracing/mutex.go#L50.

Is this something we should fix in this PR?

Reply:
looks fine as is. Level is not meaningful in the span logs, the only reason to transfer it when converting real logs is to keep data fidelity.